docs/source/devguides/supporting/examples/index.rst

```diff
@@ -6,4 +6,11 @@ Agents
     :glob:
     :maxdepth: 2
 
-    *
+    Example-Agents.rst
```

Comment:
I believe the * does the exact same thing as what you have here...though perhaps not exactly the same order.